### PR TITLE
Fix the EnZh task

### LINKED ISSUE #315: problem TranslateEnzhWmt8k
I run the Chinese-to-English translation task as the default set, the "t2t-decoder" scripts can not translate Chinese segment into English sentence, but only outputs some short Chinese sentence:

INFO:tensorflow:Inference results INPUT: 雇主们 希望 继续 积极 探讨 其他 重要 话题 —— 如 部署 、 培训 的 灵活性 、 针对 那些 在 职业 生涯 中断 后重 上岗 之 人 的 其他 培训 、 培训 成本 、 对 课程 提纲 的 共同 认知 、 进修 假 和 医疗 领域 的 性别 工资 差异 ， 这些 说明 了 雇主 、 英格兰 健康 教育 组织 ( Health Education England ) 和 卫生部 对 11月 、 2月 及 5月 与 BMA 达成 的 协议 很 重视 。
INFO:tensorflow:Inference results OUTPUT: 这 是 一 个 错误 。
INFO:tensorflow:Inference results INPUT: 在 该 法 院本 周 的 听证会 上 ， 加莱 政府 辩称 法国 政府 应 有 权 拆除 该 难民营 内 总计 72 间 临时 搭建 的 商店 。 民众 在 难民营 中的 这些 商店 里 销售 从 香烟 到 能量 饮料 在内 的 商品 、 提供 理发 服务 ， 还 在 临时 搭建 的 餐馆 中 以 50 美分 （ 40 便士 ） 的 价格 卖茶 或 以 几 欧 元 的 价格 售卖 基本 餐点 。
INFO:tensorflow:Inference results OUTPUT: 但 这 一 比例 并非 如此 。

 although the outputs of "t2t-datagen" and "t2t-trainer" scripts seems fine:

INFO:tensorflow:Saving dict for global step 250000: global_step = 250000, loss = 1.29398, metrics-translate_enzh_wmt8k/accuracy = 0.622824, metrics-translate_enzh_wmt8k/accuracy_per_sequence = 0.0, metrics-translate_enzh_wmt8k/accuracy_top5 = 0.892174, metrics-translate_enzh_wmt8k/approx_bleu_score = 0.452572, metrics-translate_enzh_wmt8k/neg_log_perplexity = -1.56626, metrics-translate_enzh_wmt8k/rouge_2_fscore = 0.756268, metrics-translate_enzh_wmt8k/rouge_L_fscore = 0.538652

I have two question:
(1) Why the Chinese-to-English translation task do not work, and the problem name "TranslateEnzhWmt8k" seems to mean translating English to Chinese. 
(2) Why not split the Chinese sentence into Chinese words before training and decoding~

Thanks in advance!

### PR DESCRIPTION
**Pull request by vince62s**
@lukaszkaiser @martinpopel 
This is to close #315 #347 and #380 

Make sure this is correct.
But people can't expect good results with this toyset. I added a comment on this.

### CODE REVIEW

File: tensor2tensor/data_generators/translate_enzh.py [hunk 1]
```diff
@@ -61,16 +63,16 @@ def num_shards(self):
 
   @property
   def source_vocab_name(self):
-    return "vocab.zhen-zh.%d" % self.targeted_vocab_size
+    return "vocab.en-zh-en.%d" % self.targeted_vocab_size
```
Comment (martinpopel) [inline tensor2tensor/data_generators/translate_enzh.py]: vocab.enzh-en

**Comment by vince62s** [comment]
thanks.
no clue why travis did not like my typo fix ....
